packages/manager/apps/hub/src/app.module.js

```diff
@@ -64,6 +66,11 @@ angular
   .config(
     /* @ngInject */ ($locationProvider) => $locationProvider.hashPrefix(''),
   )
+  .config(
+    /* @ngInject */ (ovhFeatureFlippingProvider) => {
+      ovhFeatureFlippingProvider.setApplicationName('hub');
```

Comment:
**question:** Does the application's name should reflect the folder name (here `hub`) instead of `dashboard` for instance?

Same could be applied to following list:
- `dedicated` => `bare-metal-cloud`
- `web` => `web-cloud`

Or it doesn't have sense?

My 2cts, `hub` sounds more like internal codename for a given `application`.

Reply:
`dedicated` => `bare-metal-cloud` && `hosted-private-cloud` ? and `cloud` is also `bare-metal-cloud`.

IMHO we should use internal codename (`hub`, `dedicated` etc) as it's for "internal" purpose. 
It's an application name, but the concept behind is to have a `uniqid` for each application, no matter where  it's hosted or how it's named for customers (knowing that these two parameters can change).

Reply:
Hello @antleblanc . The app name need not match the folder name. Hence, we can name the apps the way you have mentioned. But we will need to decide on the app name that we would like to use. For example, we have `bare-metal-cloud` and `hosted-private-cloud` under dedicated app. Please let me know your thoughts :)

Reply:
As the `name` can evolved based on how we want to expose it from a customer point of view, we can stick with folder name as an identifier to the application name as @cbourgois mentioned.